docs/_includes/star-rating.html

```diff
@@ -36,23 +36,23 @@ <h3>Method 2: Independent Stars</h3>
             <li>
                 <span>{{ rating }}</span>
                 <div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="{{ rating }}">
-                    <svg class="star-rating-icon" aria-hidden="true" focusable="false">
+                    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
```

Comment:
@ianmcburnie had originally asked that I use `star-rating-icon` for this class. Is this fine?

Reply:
It's fine now because we've lost the two additional classes under each icon.

```
<div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="2-5">
    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
        <use xlink:href="static/common/icons.svg#icon-star-dynamic"></use>
    </svg>

    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
        <use xlink:href="static/common/icons.svg#icon-star-dynamic"></use>
    </svg>

    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
        <use xlink:href="static/common/icons.svg#icon-star-dynamic"></use>
    </svg>

    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
        <use xlink:href="static/common/icons.svg#icon-star-dynamic"></use>
    </svg>

    <svg class="star-rating__icon" aria-hidden="true" focusable="false">
        <use xlink:href="static/common/icons.svg#icon-star-dynamic"></use>
    </svg>
</div>
```